BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -194,10 +194,10 @@
     <% t.edit_cell(:container_index_location, :header => "Index of the container image") do |f| %>
       <%= f.text_field :container_index_location %>
       <div class="field_explanation">
-        The index (url) of the container image in which the docker or singularity container is
+        The index (url) of the container image in which the docker or Apptainer/Singularity container is
         accessible through.
         Examples for Docker are: quay.io, index.docker.io (default).
-        Examples for Singularity are: docker:&sol;&sol;, shub:&sol;&sol; (default).
+        Examples for Apptainer/Singularity are: docker:&sol;&sol;, shub:&sol;&sol; (default).
```

Comment:
Huh, can you replace the `&sol;` with a single `/`, after all it's not a special character in HTML.